packages/app-page-builder/src/editor/plugins/elementSettings/save/SaveAction.tsx

```diff
@@ -82,46 +63,11 @@ const SaveAction: React.FC = ({ children }) => {
         const pbElement = (await getElementTree({ element })) as PbElement;
         formData.content = pluginOnSave(removeElementId(pbElement));
 
-        const meta = await getDataURLImageDimensions(formData.preview);
-        const blob = dataURLtoBlob(formData.preview);
-        blob.name = "pb-editor-page-element-" + element.id + ".png";
-
-        const fileUploaderPlugin = plugins.byName<FileUploaderPlugin>("file-uploader");
-        /**
-         * We break the method because it would break if there is no fileUploaderPlugin.
-         */
-        if (!fileUploaderPlugin) {
-            return;
-        }
-        const previewImage = await fileUploaderPlugin.upload(blob, { apolloClient: client });
-
-        const createFile: FileInput = {
-            ...previewImage,
-            tags: [],
-            meta: {
-                ...meta,
-                private: true
-            }
+        formData.preview = {
```

Comment:
Do we need this on the backend? 🤔 

I think not, maybe I'm wrong?

Reply:
This question still remains.

Reply:
Maybe we don't. But I need to spend a little bit more time on this question

Reply:
@adrians5j I have removed preview form the API, as we no longer need it